experimental/dds/tree2/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -63,26 +64,38 @@ export class SchemaSummarizer implements Summarizable {
 		fullTree?: boolean,
 		trackState?: boolean,
 		telemetryContext?: ITelemetryContext,
+		incrementalSummaryContext?: IExperimentalIncrementalSummaryContext | undefined,
 	): ISummaryTreeWithStats {
-		// Currently no Fluid handles are used, so just use JSON.stringify.
-		const dataString = JSON.stringify(this.codec.encode(this.schema));
-		return createSingleBlobSummary(schemaStringKey, dataString);
+		const builder = new SummaryTreeBuilder();
+		if (
+			incrementalSummaryContext !== undefined &&
+			this.schemaIndexLastChangedSeq !== undefined &&
+			incrementalSummaryContext.latestSummarySequenceNumber >= this.schemaIndexLastChangedSeq
+		) {
+			builder.addHandle(
+				schemaStringKey,
+				SummaryType.Blob,
+				`${incrementalSummaryContext.summaryPath}/indexes/Schema/${schemaStringKey}`,
```

Comment:
```suggestion
				`${incrementalSummaryContext.summaryPath}/indexes/${SchemaSummarizer.key}/${schemaStringKey}`,
```

Ideally we'd encapsulate the full path here so that it didn't need to know about `indexes` as well, but AFAICT that's basically the same TODO in the incremental summary APIs today so not too worried about making things perfect now.

Reply:
updated path to use the `SchemaSummarizer` key